src/style.css

```diff
@@ -1,3 +1,74 @@
 body {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Roboto', sans-serif;
   margin: 0;
+  background-color: aliceblue;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+/* stylelint-disable-next-line rule-empty-line-before */
+.header {
+  background-color: #fff;
+  width: 100%;
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.image {
+  height: 40px;
+  width: 40px;
+  background-color: #fff;
+  margin-left: 50px;
+}
+
+.nav {
+  display: flex;
+  text-decoration: none;
```

Comment:
The 'text-decoration: none;' property is applied to the '.nav' class, which is not necessary as it should be applied to the 'a' tags directly. Consider removing it from here to avoid redundancy.